core/src/test/java/com/amplifyframework/core/model/query/predicate/OperatorTest.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.core.model.query.predicate;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Tests operator evaluation.
+ */
+@SuppressWarnings("MagicNumber")
+public final class OperatorTest {
+
+    /**
+     * Test the accuracy of Equal operator evaluation.
+     */
+    @Test
+    public void testEqualOperator() {
+        final EqualQueryOperator numericalOperator = new EqualQueryOperator(123);
+
+        Assert.assertTrue(numericalOperator.evaluate(123));
+        Assert.assertFalse(numericalOperator.evaluate(1234));
+        Assert.assertFalse(numericalOperator.evaluate("123"));
```

Comment:
Most of the request of the project is using `import static` for the jUnit and Mockito test methods. For all unit tests, can you switch these to just `assertTrue(...)` and `assertFalse(...)`, etc.?

Reply:
Will do